com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -121,41 +121,41 @@ internal void InternalMessagesSendAndFlush(bool isListening)
                 switch (queueItem.QueueItemType)
                 {
                     case RpcQueueContainer.QueueItemType.CreateObject:
-                    {
-                        if (isListening)
                         {
-                            foreach (ulong clientId in queueItem.ClientNetworkIds)
+                            if (isListening)
                             {
-                                InternalMessageSender.Send(clientId, NetworkConstants.ADD_OBJECT, queueItem.NetworkChannel, PoolStream);
+                                foreach (ulong clientId in queueItem.ClientNetworkIds)
+                                {
+                                    InternalMessageSender.Send(clientId, NetworkConstants.ADD_OBJECT, queueItem.NetworkChannel, PoolStream);
+                                }
+
+                                PerformanceDataManager.Increment(ProfilerConstants.RpcSent, queueItem.ClientNetworkIds.Length);
+                                ProfilerStatManager.RpcsSent.Record(queueItem.ClientNetworkIds.Length);
+                                break;
```

Comment:
This breaks the logic of this switch statements.  It added  the performance stuff within the if(isListening) scope.  If it isn't listening then it still calls the performance counters.  **(i.e. it changed the behavior of the code)**

Reply:
nah, both here and the other occurrence below are simply git-diff going crazy.

the code still flows exactly the same:
- `if (isListening)`
  - `foreach (ulong clientId in queueItem.ClientNetworkIds)`
    - `InternalMessageSender.Send(clientId...)`
  - `PerformanceDataManager.Increment(...);`
  - `ProfilerStatManager.RpcsSent.Record(...);`
  - `break`

Reply:
Ahhh